tests/dummy/app/index.html

```diff
@@ -9,16 +9,16 @@
 
     {{content-for "head"}}
 
-    <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
-    <link rel="stylesheet" href="{{rootURL}}assets/dummy.css">
+    <link rel="stylesheet" href="assets/vendor.css">
+    <link rel="stylesheet" href="assets/dummy.css">
```

Comment:
Basically, you add this to the config/environment file:

https://github.com/elwayman02/ember-user-activity/blob/master/tests/dummy/config/environment.js#L43-L44